RELEASE.md

```diff
@@ -14,7 +14,6 @@
 
 - [ ] Make a new branch for the update
 - [ ] Write a blog post about the new release and put it in `/docs/_posts`
-- [ ] If necessary to preserve previous version's docs, rename `dist/reference.html` to `dist/reference-X.Y.Z.html` and add it to the list in `docs/reference-versions.html`
```

Comment:
There should be a new step to add a link from the web archiv

Reply:
I'm not so sure, it depends on how important you consider old versions to be. One of the reasons for deprecating the old docs was to simplify the release process.

Let's hear what others think.

Reply:
@Malvoz the second quote sounds good to me! Let's get this in.